docs/error_knowledge_base.md

```diff
@@ -418,14 +419,77 @@ See also: [Submitting a Package](how_to_add_packages.md#submitting-a-package).
 Consider adding the following code:
+
 ```python
 class SomeRecipe(ConanFile):
     ...
 
     short_paths = True
 ```
 
-See also: (Maximum Path Length Limitation)[https://docs.microsoft.com/en-us/windows/win32/fileio/maximum-file-path-limitation?tabs=cmd].
+See also: [Maximum Path Length Limitation](https://docs.microsoft.com/en-us/windows/win32/fileio/maximum-file-path-limitation?tabs=cmd).
+
+#### **<a name="KB-H068">#KB-H068</a>: "TEST_TYPE MANAGEMENT"**
+
+In Conan 2.0, see [migration guide](https://docs.conan.io/en/latest/migrating_to_2.0/recipes.html#changes-in-the-test-package-recipe),
+the `test_package/conanfile.py` needs to declare the requirement being tested explicitly. To be prepared you
+have to set the attribute `test_type="explicit"` (this will be ignored in 2.0) to make Conan activate the explicit
+mode, then declaring the requirement using the `self.tested_reference_str` that contains the reference being tested.
+
+#### **<a name="KB-H069">#KB-H069</a>: "TEST PACKAGE - NO DEFAULT OPTIONS"**
+
+This is to ensure the exact package that is built and uploaded is tested against. When `options` of `default_options` are modified in a
+`test_package` it can possibly result in the graph being modified. The objective is to enforce quality of the packages and to avoid confusing
+"missing packages" errors.
+
+#### **<a name="KB-H070">#KB-H070</a>: "MANDATORY SETTINGS"**
+
+> :information_source: This rule was put in place as it was deemed safe for evaluation however there is room for improvement
+
+ConanCenter operates is profiles with a predefined list of settings, all of these much be present in the recipe to ensure `package_id`s are
+computed correctly. Some libraries, for instance header-only, do not require all the settings and those should be deleted from the `package_id`.
+This approach ensure consistency and reduces the learning curve.
+
+Recipes should include:
+
+```python
+class SomeRecipe(ConanFile):
+    ...
+
+    settings = "os", "compiler", "build_type", "arch"
+```
+
+- For header-only recipes ([example](https://github.com/conan-io/conan-center-index/blob/3a773e2d69ada3bd931252c43a48daf636ddfe87/recipes/eigen/all/conanfile.py#L35-L36)):
+
+    ```python
+        def package_id(self):
+            self.info.header_only()
+    ```
+
+- For "build script" recipes ([example](https://github.com/conan-io/conan-center-index/blob/e1b9759bfc2b4625e9ba796499f743ca588d7866/recipes/automake/all/conanfile.py#L47-L50)):
+
+    ```python
+        def package_id(self):
+            del self.info.settings.arch
+            del self.info.settings.compiler
+            del self.info.settings.build_type
+    ```
+    
+- For "tool" recipes ([example](https://github.com/conan-io/conan-center-index/blob/e604534bbe0ef56bdb1f8513b83404eff02aebc8/recipes/cmake/3.x.x/conanfile.py#L104-L105)):
+
+    ```python
+        def package_id(self):
+            del self.info.settings.compiler
+    ```       
```

Comment:
Same here, explain what a _tool_ recipe is: provides binaries,... and maybe link to FAQ (or from FAQ to here) to know the reason why we keep `build_type`.

Reply:
I found something very similar in 
https://github.com/conan-io/conan-center-index/blob/92085e1ba5339043e01c4c9e67fd2dce961719af/docs/packaging_policy.md#settings